velox/exec/Spiller.cpp

```diff
@@ -733,9 +747,10 @@ void Spiller::clearNonSpillingRuns() {
 
 std::string Spiller::toString() const {
   return fmt::format(
-      "{}\t{}\tMAX_PARTITIONS:{}\tFINALIZED:{}",
+      "{}\t{}\tCONTAINER_COLUMNS:{}\tMAX_PARTITIONS:{}\tFINALIZED:{}",
       typeName(type_),
       rowType_->toString(),
+      container_->columnTypes().size(),
```

Comment:
We can log out columnTypes() when check fails explicitly

Reply:
I thought about it, I ended up deciding that column types don't really matter here.
If we have different number of children, then two sets of types are going out of sync somewhere - that's all we need to know.